draft-ietf-tls-external-psk-importer.md

```diff
@@ -87,6 +87,47 @@ a tuple of (Base Key, External Identity, KDF). The associated KDF (and hash func
 - External Identity: The identity of an EPSK.
 - Imported Identity: The identity of a PSK as sent on the wire.
 
+#Binder Key
+To prevent PSK Importers from being confused with standard OOB PSKs we change the label used in the computation of the PSK binder key.
+In TLS 1.3 the  PSK binder key computation is defined as follows:
+
+                 0
+                 |
+                 v
+       PSK ->  HKDF-Extract = Early Secret
+                 |
+                 +-----> Derive-Secret(., "ext binder" | "res binder", "")
+                 |                     = binder_key
+
+We replace the string "ext binder" with "imp binder".
+This means the binder key is now computed as follows:
+
+                 0
+                 |
+                 v
+       PSK ->  HKDF-Extract = Early Secret
+                 |
+                 +-----> Derive-Secret(., "ext binder" 
+                                        | "res binder" 
+                                        | "imp binder", "")
+                 |                     = binder_key
+
+This change should not affect any of the securtiy properties or security proofs previously established.
+The only change that is not a previously valid TLS 1.3 trace we mostly focus our attention on the change to the `binder_key` computation.
```

Comment:
This sentence doesn't parse. Can you please rephrase?

Reply:
Ugh, this was a copy paste error. It should read
"The only change that is not a previously valid TLS 1.3 trace in the change to the `binder_key` computation. We thus mostly focus our attention there."